### ISSUE TITLE
[Feature Request] Be able to make new lines despite Enter-As-Send via Ctrl-Enter (or Shift-Enter)

### ISSUE THREAD

**Issue by KeronCyst** [forkgram/TelegramAndroid#84]
When typing with a physical connected keyboard or sending keystrokes via adb. Thanks!!

**Comment by 23rd** [comment]
Can you check the 8.8.2.0 version?

**Comment by ArenaL5** [comment]
(Checking for stale issues... I'd never tried sending keystrokes with `adb`, it's kinda fun! In a novelty toy kind of way)

TL;DR: I'm using Telegram 11.9.3.0 on crDroid 11.6, based on Android 15. **I can confirm this has been added.**

Tests from ADB
------

I have the app open on my Saved messages, and the text field on focus.

Note that, nowadays just pressing the Enter key in a virtual keyboard on the Forkgram app creates a new line; it doesn't automatically send the message like in desktop clients.

### Trying to directly type a text from adb shell
```bash
$ adb shell input text "Multiline
text"
/system/bin/sh: text: inaccessible or not found
``` 
Only `Multiline` is typed. The message is not sent. (I'm pretty sure there is some funky newline reinterpretation when running a command this way)

### Opening a shell, then using `input text`
Sending a text with `input text`, like this:
```bash
$ adb shell
ginkgo:/ $ input text "Multiline
> text"
```
Both lines are typed in a single message. The message is not sent.

### Sending singular keystrokes
First I'm going to open a shell,
```bash
$ adb shell
```

Then I'm going to type some single events:
```bash
input keyevent KEYCODE_A
input keyevent KEYCODE_ENTER
input keyevent KEYCODE_B
input keycombination KEYCODE_SHIFT_LEFT KEYCODE_ENTER
input keyevent KEYCODE_C
input keycombination KEYCODE_CTRL_LEFT KEYCODE_ENTER

```

The result is that it sends a single three-line message:
> a
> b
> c

Tests on the phone
-----

I'm also using Unexpected Keyboard (a virtual keyboard on the phone, that allows me to sent keycombos with Ctrl and Shift like that without a computer) and the results are identical.
